Tiltfile

```diff
@@ -333,14 +342,14 @@ def deploy_worker_templates(template, substitutions):
     # programmatically define any remaining vars
     # "windows" can not be for cluster name because it sets the dns to trademarked name during reconciliation
     substitutions = {
-        "AZURE_LOCATION": "eastus",
+        "AZURE_LOCATION": os.getenv("AZURE_LOCATION", "eastus"),
         "AZURE_VNET_NAME": "${CLUSTER_NAME}-vnet",
         "AZURE_RESOURCE_GROUP": "${CLUSTER_NAME}-rg",
-        "CONTROL_PLANE_MACHINE_COUNT": "1",
-        "KUBERNETES_VERSION": settings.get("kubernetes_version"),
-        "AZURE_CONTROL_PLANE_MACHINE_TYPE": "Standard_B2s",
-        "WORKER_MACHINE_COUNT": "2",
-        "AZURE_NODE_MACHINE_TYPE": "Standard_B2s",
+        "CONTROL_PLANE_MACHINE_COUNT": os.getenv("CONTROL_PLANE_MACHINE_COUNT", "1"),
+        "KUBERNETES_VERSION": os.getenv("KUBERNETES_VERSION", settings.get("kubernetes_version")),
+        "AZURE_CONTROL_PLANE_MACHINE_TYPE": os.getenv("AZURE_CONTROL_PLANE_MACHINE_TYPE", "Standard_B2s"),
+        "WORKER_MACHINE_COUNT": os.getenv("WORKER_MACHINE_COUNT", "2"),
+        "AZURE_NODE_MACHINE_TYPE": os.getenv("AZURE_NODE_MACHINE_TYPE", "Standard_B2s"),
```

Comment:
I am not too opinionated about this change, but would like to have the flexibility to set some of the variables. What do you think?